indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -210,9 +211,17 @@ public boolean run()
 
       boolean success = job.waitForCompletion(true);
 
-      Counter invalidRowCount = job.getCounters()
-                                   .findCounter(HadoopDruidIndexerConfig.IndexJobCounters.INVALID_ROW_COUNTER);
-      jobStats.setInvalidRowCount(invalidRowCount.getValue());
+      Counters counters = job.getCounters();
+      if (counters == null) {
```

Comment:
Why'd you need to add these? Why would the counters not be set?

Reply:
It happened on a task failure I was testing the ParseExceptions, didn't look further into why the job counters weren't set